report-flat/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "report-flat",
+  "version": "1.0.0",
+  "description": "aet-printable-report",
+  "author": "subiron <misiewicz.tomasz@gmail.com>",
+  "private": true,
+  "scripts": {
+    "dev": "node build/dev-server.js",
+    "build": "node build/build.js",
+    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
+    "e2e": "node test/e2e/runner.js",
+    "test": "npm run unit && npm run e2e",
+    "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs"
+  },
+  "dependencies": {
+    "vue": "^2.1.0"
+  },
+  "devDependencies": {
+    "autoprefixer": "^6.4.0",
```

Comment:
Do we need all that dependencies are we using them, or this is some sort of boilerplate leftover?